packages/core/admin/admin/src/pages/MarketplacePage/components/OfflineLayout/index.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import { useIntl } from 'react-intl';
+import { Box } from '@strapi/design-system/Box';
+import { Flex } from '@strapi/design-system/Flex';
+import { Typography } from '@strapi/design-system/Typography';
+import { Layout } from '@strapi/design-system/Layout';
+import { Main } from '@strapi/design-system/Main';
+import PageHeader from '../PageHeader';
+import offlineCloud from '../../../../assets/images/icon_offline-cloud.svg';
+
+const OfflineLayout = () => {
+  const { formatMessage } = useIntl();
+
+  return (
+    <Layout>
+      <Main>
+        <PageHeader isOnline={false} />
+        <Flex
+          width="100%"
+          direction="column"
+          alignItems="center"
+          justifyContent="center"
+          style={{ paddingTop: '120px' }}
```

Comment:
`paddingTop={pxToRem(120)}` doesn't work?

Reply:
I'll check I haven't edited the code, but I don't see why it wouldn't